dts/bindings/interrupt-controller/st,stm32g0-exti.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2023, STMicroelectronics
+
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+        STM32 controller
+        This compatible stands for all interrupt-controller blocks for the
+        STM32C0/G0/L5/mp/u5.
+
+compatible: "st,stm32g0-exti"
+
+include: [base.yaml, interrupt-controller.yaml]
```

Comment:
Shouldn't we include [st,stm32-exti.yaml] instead, like it is done for example for uart/usart, to remove the following compilation error:
devicetree error: 'num-lines' appears in /soc/interrupt-controller@46022000 in /__w/zephyr/zephyr/twister-out/b_u585i_iot02a/tests/drivers/clock_control/stm32_clock_configuration/stm32u5_core/drivers.stm32_clock_configuration.u5.sysclksrc_pll_hsi_40/zephyr/zephyr.dts.pre, but is not declared in 'properties:' in /__w/zephyr/zephyr/dts/bindings/interrupt-controller/st,stm32g0-exti.yaml

Reply:
OK